recipes/re2/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "re2" %}
+{% set version = "2016.11.01" %}
+{% set dashversion = version.replace('.', '-') %}
+{% set sha256 = "01ee949f03e1c4057dc533cf139f967fb1b427015769d53b9ee07757631e9669" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/google/{{ name }}/archive/{{ dashversion }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - win-cmakelists.patch  # [win]
+
+build:
+  number: 0
+  skip: True  # [not py35]
```

Comment:
So `conda-build-all` is used in `staged-recipes` and it runs using [matrix conditions]( https://github.com/conda-forge/staged-recipes/blob/fa6d6c913cc49982cd39d9601162315f88e144ce/appveyor.yml#L99 ) that effectively force a range of `python` versions to be tried anyways.

While the feedstocks have the same [matrix conditions]( https://github.com/conda-forge/conda-smithy/blob/4f10b7d47b08b9e98b216be2d31bfe3d14673fb7/conda_smithy/configure_feedstock.py#L405 ) used by `conda-build-all` to inform `conda-smithy`, they are used to determine what [actually cases]( https://github.com/conda-forge/conda-smithy/blob/4f10b7d47b08b9e98b216be2d31bfe3d14673fb7/conda_smithy/configure_feedstock.py#L279-L280 ) will really be built for to [fill out the template]( https://github.com/conda-forge/conda-smithy/blob/4f10b7d47b08b9e98b216be2d31bfe3d14673fb7/conda_smithy/configure_feedstock.py#L322-L324 ). As there never was a Python 3.5 case, we find the CI is [dropped]( https://github.com/conda-forge/conda-smithy/blob/4f10b7d47b08b9e98b216be2d31bfe3d14673fb7/conda_smithy/configure_feedstock.py#L289-L293 ).